core/src/main/java/com/linecorp/armeria/common/util/BlockingTaskExecutorBuilder.java

```diff
@@ -136,6 +136,9 @@ public BlockingTaskExecutor build() {
         final ScheduledThreadPoolExecutor scheduledThreadPoolExecutor = new ScheduledThreadPoolExecutor(
                 numThreads, threadFactory);
         scheduledThreadPoolExecutor.setKeepAliveTime(keepAliveTimeMillis, TimeUnit.MILLISECONDS);
+        if (keepAliveTimeMillis > 0) {
+            scheduledThreadPoolExecutor.allowCoreThreadTimeOut(true);
+        }
```

Comment:
```suggestion
        if (keepAliveTimeMillis > 0) {
            scheduledThreadPoolExecutor.setKeepAliveTime(keepAliveTimeMillis, TimeUnit.MILLISECONDS);
            scheduledThreadPoolExecutor.allowCoreThreadTimeOut(true);
        }
```

Reply:
um..I think it doesn't work when someone want to set keepAliveTime to 0.
In this case, the result will be `keepAliveTime = 10L` and `allowCoreThreadTimeOut = false`
Am I mistaken..?😢